src/common/UserAvatar.js

```diff
@@ -41,7 +41,9 @@ export default class UserAvatar extends PureComponent<Props> {
         <Touchable onPress={onPress} style={style}>
           <ImageBackground
             style={style}
-            source={{ uri: avatarUrl }}
+            source={{
+              uri: avatarUrl.get(size * PixelRatio.get()).toString(),
```

Comment:
I think this will often be not an integer -- as mentioned [in our layout doc](https://github.com/zulip/zulip-mobile/blob/998947d8073f4b339d8086824e5cfc8703577c33/docs/background/layout.md#display-size-logical-pixels), possible values for the ratio include 2.61 or 4.5. Particularly with a ratio like 2.61, this could wind up with a value like 62.64.

I think the one place in the avatar code where that's potentially a problem may be when we stick a size on a Gravatar URL -- other than that, I think we just compare to the threshold of 100.

But it may be cleanest for these callers to have responsibility for providing an integer, because that actually coincides with using what's probably a slightly nicer bit of the `PixelRatio` API here anyway: `avatarUrl.get(PixelRatio.getPixelSizeForLayoutSize(size))`.